### DISCUSSION TITLE
Does Renovate close PRs with invalid configuration changes?

### DISCUSSION THREAD

**Discussion by tomkerkhove** [renovatebot/renovate#12885] [Request Help] [answered]
Does Renovate close PRs with invalid configuration changes?

I've been fighting with Renovate on a PR (https://github.com/kedacore/keda/pull/2361) and think this is the reason, but a message on the PR would be nice to help point out that the config is invalid.

**Comment by viceice** [comment, marked as answer]
you schouldn't use renovate branch prefix for other branches.

**Reply by viceice** [reply]
https://docs.renovatebot.com/configuration-options/#branchprefix
https://docs.renovatebot.com/configuration-options/#prunestalebranches

renovate will delete all branches starting with `branchPrefix`

**Reply by tomkerkhove** [reply]
Good to know, thanks.
